src/math/curve.js

```diff
@@ -23,12 +23,13 @@ import { CurveEvaluator } from './curve-evaluator.js';
  *
  * @example
  * var curve = new pc.Curve(
- * [
- *     [0, 0],
- *     [0.33, 2],
- *     [0.66, 2.6],
- *     [1, 3]
- * ]);
+ *     [
```

Comment:
CurveSet has:
```javascript
new pc.CurveSet([
    [
        [0, 0],
        [0, 0],
    ],
    [
        [0, 0],
        [0, 0]
    ]
]);
```
So it clearly shows it is array of groups of tuples.
But for Curve, it should not look similar, as there is one depth less, so it probably should be:
```javascript
new pc.Curve([
    [0, 0],
    [0, 0]
])
```

Reply:
Ah, good shout. That explains why the linter was complaining